web-services/common-util/src/test/java/datawave/security/util/AuthorizationsUtilTest.java

```diff
@@ -49,6 +49,15 @@ public void initialize() {
         DatawaveUser p2 = new DatawaveUser(p2dn, UserType.SERVER, Sets.newHashSet("A", "F", "G"), null, null, System.currentTimeMillis());
         
         principal = new DatawavePrincipal(Lists.newArrayList(user, p1, p2));
+        
+        Collection<Collection<String>> cbAuths = new HashSet<>();
```

Comment:
what are we trying to do here?  these cbAuths are not used anywhere else.

Reply:
Doh! Thanks. Leftover cruft from my testing. Will remove